common/lib/xmodule/xmodule/discussion_module.py

```diff
@@ -42,7 +44,20 @@ class DiscussionFields(object):
     sort_key = String(scope=Scope.settings)
 
 
+def has_permission(user, permission, course_id):
+    """
+    Copied from django_comment_client/permissions.py because I can't import
+    that file from here. It causes the xmodule_assets command to fail.
+    """
+    return any(role.has_permission(permission)
+               for role in user.roles.filter(course_id=course_id))
+
+
+@XBlock.wants('user')
```

Comment:
This is a refactor, so I was trying to match the existing implementation as closely as possible.